velox/exec/tests/JoinFuzzer.cpp

```diff
@@ -164,19 +233,18 @@ bool isDone(size_t i, T startTime) {
   return i >= FLAGS_steps;
 }
 
-core::JoinType JoinFuzzer::pickJoinType() {
+void JoinFuzzer::pickJoinType() {
   // Right joins are tested by flipping sides of the left joins.
   static std::vector<core::JoinType> kJoinTypes = {
       core::JoinType::kInner,
       core::JoinType::kLeft,
       core::JoinType::kFull,
       core::JoinType::kLeftSemiFilter,
       core::JoinType::kLeftSemiProject,
-      core::JoinType::kAnti,
-  };
+      core::JoinType::kAnti};
 
-  size_t idx = randInt(0, kJoinTypes.size() - 1);
-  return kJoinTypes[idx];
+  const size_t idx = randInt(0, kJoinTypes.size() - 1);
+  joinType_ = kJoinTypes[idx];
```

Comment:
Why is this change? It feels hard to maintain a large list of member variables with different methods changing some of these.